pkg/kdk/config.go

```diff
@@ -16,16 +16,21 @@ package kdk
 
 import (
 	"context"
+	"github.com/cisco-sso/kdk/pkg/utils"
```

Comment:
Should: Move the github dep down to the non-core libs list which starts at line 25.

Reply:
good catch.
fixed.